libs/api/domains/cms/src/lib/search/importers/article.service.ts

```diff
@@ -31,16 +31,20 @@ export class ArticleSyncService implements CmsSyncProvider<IArticle> {
     // only process articles that we consider not to be empty and dont have circular structures
     return entries.reduce((processedEntries: IArticle[], entry: Entry<any>) => {
       if (this.validateArticle(entry)) {
-        // remove nested related articles from releated articles
-        const relatedArticles = (entry.fields.relatedArticles ?? []).map(
-          ({
-            sys,
-            fields: { relatedArticles, ...prunedRelatedArticlesFields },
-          }) => ({
-            sys,
-            fields: prunedRelatedArticlesFields,
-          }),
-        )
+        // remove nested related articles from related articles
+        const relatedArticles = (entry.fields.relatedArticles || [])
+          .map(({ sys, fields }) => {
+            // handle if some one deletes an article without removing reference case, this will be fixed more permanently at a later time with nested resolvers
```

Comment:
```suggestion
            // handle if someone deletes an article without removing reference case, this will be fixed more permanently at a later time with nested resolvers
```